src/specialElHandlers.js

```diff
@@ -12,11 +12,14 @@ function syncBooleanAttrProp(fromEl, toEl, name) {
 }
 
 export default {
-    /**
-     * Needed for IE. Apparently IE doesn't think that "selected" is an
-     * attribute when reading over the attributes using selectEl.attributes
-     */
     OPTION: function(fromEl, toEl) {
+        var parentNode = fromEl.parentNode;
+        if (parentNode && parentNode.nodeName === 'SELECT' && !hasAttributeNS(parentNode, null, 'multiple')) {
```

Comment:
previous code used `.toUpperCase()` every time the `.nodeName` was retrieved. I think that's a good strategy for backward compatibility.

Reply:
Done, thanks.